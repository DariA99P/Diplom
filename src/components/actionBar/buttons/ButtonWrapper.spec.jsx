import React from 'react';
import { shallow } from 'enzyme';

import ButtonWrapper from '.';

describe('<ButtonWrapper />', () => {
  let tree;
  beforeEach(() => {
    tree = shallow(<ButtonWrapper />);
  });

  describe('On rendering', () => {
    it('should render properly', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
