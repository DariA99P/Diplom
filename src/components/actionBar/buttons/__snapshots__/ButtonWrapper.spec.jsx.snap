// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonWrapper /> On rendering should render properly 1`] = `
<div>
  <Button
    block={false}
    ghost={false}
    htmlType="button"
    loading={false}
    type="primary"
  >
    <Icon
      type="book"
    />
    Save
  </Button>
</div>
`;
